code/latte/rational.cpp

```diff
@@ -200,6 +200,11 @@ void RationalNTL::canonicalize()
 		denominator *= -1;
 		numerator *= -1;
 	}//make denominator positive.
+	else if (denominator == 0) 
```

Comment:
This case happens when `denominator = 0` and `numerator = 0`. With older NTL versions, this method returned `0`, with newer ones an error occured.